images/server/install-sambacc.sh

```diff
@@ -3,18 +3,44 @@
 set -ex
 
 wheeldir="$1"
-container_json_file="$2"
 
 wheeldir=/tmp
-wheel="$(find "${wheeldir}" -type f -name 'sambacc-*.whl')" \
+find "${wheeldir}"
+wheel="$(find "${wheeldir}" -type f -name 'sambacc-*.whl')"
+rpmfile="$(find "${wheeldir}" -type f -name '*sambacc-*.noarch.rpm')"
 
-if ! [ "$(echo "$wheel" | wc -l)" = 1 ]; then
-    echo "more than one wheel file found"
+rpmcount="$(echo "$rpmfile" | grep -c rpm)"
```

Comment:
Same comment as above, regrading grep exit with 1 which causes the entire script to fail due to `set -ex` at line 3.